PHPCI/Languages/lang.en.php

```diff
@@ -113,6 +113,7 @@
                                 (if you cannot add a phpci.yml file in the project repository)',
     'default_branch' => 'Default branch name',
     'allow_public_status' => 'Enable public status page and image for this project?',
+    'archived' => 'Archived',
```

Comment:
I can't understand why you don't use .po/.mo file types for translations, benefits in .pot file that contains all original strings and any LANG.po files can be updated from .pot file.